includes/ApplHandlingByStr.php

```diff
@@ -96,6 +96,7 @@ function findPending( $stat_str )
 		
 		if( $str[1] == 'P' )
 		{
+		echo $usr;
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Line indented incorrectly; expected at least 12 spaces, found 8

<!-- Comment by [SideCI](https://sideci.com) -->